### PR TITLE
fix(userspace/libscap): fallback at considering cpus online when NUMA topology information is missing

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area libscap-engine-bpf

**Does this PR require a change in the driver versions?**

**What this PR does / why we need it**:

This fixes running Falco with eBPF on raspberry PI devices.
See https://github.com/kubernetes/kubernetes/issues/95039#issuecomment-940976183 and https://github.com/falcosecurity/falco/issues/2246.

**Which issue(s) this PR fixes**:

https://github.com/falcosecurity/falco/issues/2246

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: userspace/libscap/engine/bpf/scap_bpf.c [hunk 1]
```diff
@@ -1504,20 +1504,33 @@ int32_t scap_bpf_load(
...
+				// This is useful for example for raspPi devices.
+				// See: https://github.com/kubernetes/kubernetes/issues/95039
+				snprintf(filename, sizeof(filename), "/sys/devices/system/cpu/cpu%d/", j);
+				if (access(filename, F_OK) == 0)
+				{
+					online = 1;
+				}
+				else
+				{
+					snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "can't open %sonline: %s", filename, scap_strerror_r(buf, errno));
```
Comment (Andreagit97) [inline userspace/libscap/engine/bpf/scap_bpf.c:1518]:
```suggestion
					snprintf(handle->m_lasterr, SCAP_LASTERR_SIZE, "cannot access '%s' file: %s", filename, scap_strerror_r(buf, errno));
```

Comment (Andreagit97) [reply userspace/libscap/engine/bpf/scap_bpf.c:1518]: `%sonline` is misleading and is not the actual file we are looking for here, so I would remove it... just proposed a rephrase but feel free to ignore it

**Review by Andreagit97 — COMMENTED** [review]
just a minor comment :)

File: userspace/libscap/engine/bpf/scap_bpf.c (hunk 1 above)
Comment (FedeDP) [reply userspace/libscap/engine/bpf/scap_bpf.c:1518]: Mmmh if both `/sys/devices/system/cpu/cpu%d/online` and `/sys/devices/system/cpu/cpu%d/` cannot be reached, i'd like to notify the user about the former.

**Review by Andreagit97 — APPROVED** [review]
/approve

**Review by leogr — APPROVED** [review]

**Pull request merged**